apps/web/src/pages/Applications/ApplicationWelcomePage/ApplicationWelcomePage.tsx

```diff
@@ -13,6 +13,8 @@ import { useUpdateApplicationMutation, ApplicationStep } from "~/api/generated";
 
 import { errorMessages } from "@gc-digital-talent/i18n";
 import ApplicationApi, { ApplicationPageProps } from "../ApplicationApi";
+import { PublishingGroup } from "@gc-digital-talent/graphql";
```

Comment:
This can be removed now

Reply:
😅 Fixed by [5d9ed81](https://github.com/GCTC-NTGC/gc-digital-talent/pull/6545/commits/5d9ed81013041c5c62f0f228a38bc42cf21de005)